apps/srp.c

```diff
@@ -291,7 +297,7 @@ int srp_main(int argc, char **argv)
                    "Need at least one user for options -add, -delete, -modify. \n");
         goto opthelp;
     }
-    if ((passin || passout) && argc != 1) {
+    if ((passinarg || passoutarg) && argc != 1) {
```

Comment:
Ouch. On side note there was a push for making checks for NULL pointers explicit, for example `passinarg != NULL` instead of just `passinarg`. However, there are other occurrences of implicit NULL checks in *this* module, and so it kind of falls beyond scope of this request [which is why it's a side note.]